frontend/prisma/schema.prisma

```diff
@@ -15,16 +15,46 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model User {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  email     String?  @unique
-  image     String?
-  role      Role     @default(USER)
+model Clubs {
+  cid         String         @id @default(uuid())
+  email       String         @unique
+  password    String
+  info        ClubInfo[]
+  events      Events[]
```

Comment:
just wondering, how would we be using this column? i was thinking we would be using the events table whenever we want to access events

Reply:
It's just to establish a link between Events and Clubs. This is essentially saying the events table has a many to one relationship to clubs